src/textual/drivers/linux_driver.py

```diff
@@ -146,8 +146,9 @@ def on_terminal_resize(signum, stack) -> None:
         self._enable_bracketed_paste()
 
     def _request_terminal_sync_mode_support(self):
-        self.console.file.write("\033[?2026$p")
-        self.console.file.flush()
+        if self.console._environ.get("TERM_PROGRAM", "") != "Apple_Terminal":
```

Comment:
Could you add a comment here saying why this code is here? It's the sort of thing someone else could look at 6 months from now and not understand.

Reply:
Yeah, fair enough.

Reply:
Done